editor/src/components/canvas/canvas-strategies/strategies/forced-absolute-reparent-strategies.spec.browser2.tsx

```diff
@@ -732,22 +743,10 @@ describe('Forced Absolute Reparent Strategies', () => {
             data-testid='flexchild2'
           />
         </div>
-        <div
-          style={{
-            width: 100,
-            height: 100,
-            backgroundColor: 'teal',
-            position: 'absolute',
-            left: 75,
-            top: 200,
-          }}
-          data-uid='flexchild1'
-          data-testid='flexchild1'
-        />
       </div>`),
     )
   })
-  it('Absolute to absolute on a target parent element without size', async () => {
+  it('Absolute to static on a target parent element without size', async () => {
```

Comment:
Interesting... was this change deliberate?

Reply:
if you reparent into a sizeless display:flow target parent, according to the new rules that always leads the child being converted to static. since this test is _primarily_ testing sizeless parents, and the new correct behavior is a static reparent, I changed the test to reflect that.